02week/pigLatin.js

```diff
@@ -9,9 +9,24 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  let cut = word.trim().toLowerCase();         // The word inputted will be changed to lowercase and all spaces before and after the word will be trimmed.
+  console.log(cut);
```

Comment:
@AFutureWebDev 
This is fine for now, but it's good practice to remove console.logs before making a PR.
In the real world, they are typically not committed to the main code base.